bot/paginatedmessages/paginatedinteractions.go

```diff
@@ -93,11 +95,17 @@ func createNavigationButtons(prevDisabled bool, nextDisabled bool) []discordgo.M
 	}
 }
 
-func CreatePaginatedMessage(guildID, channelID int64, initPage, maxPages int, pagerFunc PagerFunc) (*PaginatedMessage, error) {
+func CreatePaginatedMessage(guildID, channelID int64, initPage, maxPages int, pagerFunc PagerFunc, data ...*dcmd.Data) (*PaginatedMessage, error) {
 	if initPage < 1 {
 		initPage = 1
 	}
 
+	var interaction = &discordgo.Interaction{ID: 0}
+
+	if len(data) >= 1 && !reflect.ValueOf(data[0].SlashCommandTriggerData).IsNil() {
```

Comment:
Why is reflection used here? If it is truly necessary, a comment might be nice -- this pattern does not seem to be used anywhere else in the codebase -- but I suspect it is gratuitous and a direct test against `nil` would suffice.

Reply:
I forgot to remove that, it's not supposed to be anymore. It was because Discord had a bug with the response.